client/post-editor/editor-diff-viewer/index.jsx

```diff
@@ -88,7 +96,13 @@ class EditorDiffViewer extends PureComponent {
 	};
 
 	recomputeChanges = callback => {
-		const diffNodes = this.node.querySelectorAll( '.text-diff__additions, .text-diff__deletions' );
+		let selectors = '.text-diff__additions, .text-diff__deletions';
+		if ( this.windowWidth > 1040 && this.props.diffView === 'split' ) {
```

Comment:
Hmm, to avoid a magic number here, we might consider updating `isWithinBreakpoint` in `client/lib/viewport` to support the other breakpoints noted in `_breakpoints.scss`.

Reply:
I've updated `isWithBreakpoint` in https://github.com/Automattic/wp-calypso/pull/24352, and I'll integrate the change into `EditorDiffViewer` in https://github.com/Automattic/wp-calypso/pull/24355.